### DISCUSSION TITLE
經銷商詢價問題

### DISCUSSION THREAD

**Discussion by Linksoft2007** [dashingsoft/pyarmor#1901] [General]
已寫信至 貴司 pyarmor@163.com 信箱未獲回覆，
我司(友環股份有限公司) 為台灣軟體經銷商，
現我司有一客戶需求Pyarmor 集团版 1套，
請提供報價及經銷折扣，
另，線上購買系統需填入中國的稅號，請問台灣公司是否能購買及使用呢?
謝謝 !

**Comment by Linksoft2007** [comment]
請問此案有任何進展嗎? 謝謝 !

**Comment by jondy** [comment]
Pyarmor 现在不通过分销商进行销售，请直接通过 MyCommerce 或者 pyarmor.dashingsoft.com 上面的链接购买

**Comment by riiiiiiki** [comment]
Sr for comment this. How can i add version info file exe with pack onefile pyarmor. I try find doc but not found

**Comment by jondy** [comment]
Does this make sense?
https://pyarmor.readthedocs.io/en/latest/topic/repack.html#using-more-pyinstaller-options
